cmd/mdatagen/metadata-schema.yaml

```diff
@@ -130,31 +130,32 @@ tests:
 # Optional: map of metric names with the key being the metric name and value
 # being described below.
 telemetry:
-  <metric.name>:
-    # Required: whether the metric is collected by default.
-    enabled: bool
-    # Required: metric description.
-    description:
-    # Optional: extended documentation of the metric.
-    extended_documentation:
-    # Optional: warnings that will be shown to user under specified conditions.
-    warnings:
-      # A warning that will be displayed if the metric is enabled in user config.
-      # Should be used for deprecated default metrics that will be removed soon.
-      if_enabled:
-      # A warning that will be displayed if `enabled` field is not set explicitly in user config.
-      # Should be used for metrics that will be turned from default to optional or vice versa.
-      if_enabled_not_set:
-      # A warning that will be displayed if the metrics is configured by user in any way.
-      # Should be used for deprecated optional metrics that will be removed soon.
-      if_configured:
-    # Required: metric unit as defined by https://ucum.org/ucum.html.
-    unit:
-    # Required: metric type with its settings.
-    <sum|gauge|histogram>:
-      # Required: type of number data point values.
-      value_type: <int|double>
-      # Required for sum metric: whether the metric is monotonic (no negative delta values).
-      monotonic: bool
-    # Optional: array of attributes that were defined in the attributes section that are emitted by this metric.
-    attributes: [string]
+  metrics:
```

Comment:
This is really the only difference here. Everything else was just indented under `metrics`.